sdk/firstload/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "highlight.run",
-	"version": "5.2.3",
+	"version": "6.0.0",
```

Comment:
@Vadman97 do you think we should do a major version bump since this will technically break existing functionality (Intercom integration wouldn't be enabled by default)?

Reply:
probably, but what's the reasoning for having these integrations opt-in? given no other configuration is required, i'd say that the integrations should be opt-out for specific cases where a user does not want to send their analytics metadata to highlight.

Reply:
That's cool with me. @aptlin surfaced this and we thought it made more sense to make them opt-in, but I can get behind making them opt-out instead 👍 

Have any opinions on how to specify it? Maybe set `H.options.integrations.[integrationName].disabled` to `true`?

Reply:
@Vadman97 I ended up making it so you can opt out 👍

Reply:
sounds good!